usr/src/test/zfs-tests/tests/functional/cli_root/zpool_import/zpool_import_rename_001_pos.ksh

```diff
@@ -159,4 +161,13 @@ while (( i < ${#pools[*]} )); do
 	((i = i + 1))
 done
 
+VDEV_FILE=$(mktemp /tmp/tmp.XXXXXX)
+
+log_must truncate --size=128M $VDEV_FILE
```

Comment:
Can we use `mkfile -n` here instead to make the file sparse (in case we don't have 128MB in /tmp/, that will make it less likely to fail intermittently) and skip using truncate(1), which comes from external package (GNU coreutils)?